pkg/ibmvpc/vpc.go

```diff
@@ -263,10 +263,6 @@ func (fip *FloatingIP) RouterDefined(src, dst vpcmodel.Node) bool {
 		(vpcmodel.HasNode(fip.Sources(), dst) && src.IsExternal())
 }
 
-func (fip *FloatingIP) AppliedFiltersKinds() map[string]bool {
```

Comment:
No, we still need it there. 
`func (m *mockNetIntf) AppliedFiltersKinds(otherNode InternalNodeIntf) map[string]bool `

Is still needed since it is still used in the interface of `InternalNode`  (vsis in the same subnet do not need nacl).
It was only removed from Router interface